bluesky/spec_api.py

```diff
@@ -132,6 +135,9 @@ def ascan(motor, start, finish, intervals, time=None, *, md=None):
     md : dict, optional
         metadata
     """
+    if md is None:
+        md = {}
+    md = ChainMap(md, {'plan_name': 'ascan'})
```

Comment:
Why use ChainMap instead of 
```
md.update({'plan_name': 'ascan'})
```

I am positive that I just missed this discussion. Sorry for making you repeat it